models/engine/db_storage.py

```diff
@@ -78,3 +80,22 @@ def close(self):
             calls remove() on private session attribute (self.session)
         """
         self.__session.remove()
+
+    def get(self, cls, id):
+        """
+        Returns the object based on the class name and its ID
+        """
+        if cls:
+            obj_class = self.__session.query(self.CNC.get(cls)).all()
```

Comment:
No need to query here, you can simply use the `.all()` function on `self`...  Also, remember `.all()` is setup to take `cls` as an argument, if you use that argument you will not have to rewrite so much code.